src/actions/lang.ts

```diff
@@ -0,0 +1,21 @@
+import * as types from './types';
+
+export type LangActions = {
+  LANG_FI: {
```

Comment:
just fixed similar problem in master, but
we should use [types.LANG_FI]: instead of LANG_FI here

Reply:
actually I can do it after I merge the changes in :)